packages/web-components/fast-foundation/src/menu-item/menu-item.template.ts

```diff
@@ -23,8 +23,14 @@ export function menuItemTemplate<T extends FASTMenuItem>(
         @click="${(x, c) => x.handleMenuItemClick(c.event as MouseEvent)}"
         @mouseover="${(x, c) => x.handleMouseOver(c.event as MouseEvent)}"
         @mouseout="${(x, c) => x.handleMouseOut(c.event as MouseEvent)}"
-        class="${x => (x.disabled ? "disabled" : "")} ${x =>
-        x.expanded ? "expanded" : ""} ${x => `indent-${x.startColumnCount}`}"
+        class="${x =>
+            [
+                x.disabled && "disabled",
+                x.expanded && "expanded",
+                typeof x.startColumnCount === "number" && `indent-${x.startColumnCount}`,
+            ]
+                .filter(Boolean)
+                .join(" ")}"
```

Comment:
Everything but the indent should likely be removed class-wise here. Note for myself...the default menu item likely should not have a construct for "indent"...